routing/missioncontrol.go

```diff
@@ -261,12 +262,56 @@ type MissionControlPairSnapshot struct {
 // paymentResult is the information that becomes available when a payment
 // attempt completes.
 type paymentResult struct {
-	id                 uint64
-	timeFwd, timeReply time.Time
-	route              *mcRoute
-	success            bool
-	failureSourceIdx   *int
-	failure            lnwire.FailureMessage
+	id               uint64
+	timeFwd          tlv.RecordT[tlv.TlvType1, uint64]
```

Comment:
The reason we don't start at type 0 here is because of the ID which is part of the key and not the actual data, I assume? Could be worth a tiny comment.

Reply:
ah - good point. This was an oversight cause I initially had ID as type 0. But then realised it is not actually serialised. Have updated so that timeFwd is type 0 and so on 👍